src/explorer/model.ts

```diff
@@ -312,18 +337,40 @@ class PipelineNode implements TektonNode{
     return this.children.length > 0? TreeItemCollapsibleState.Expanded : TreeItemCollapsibleState.Collapsed;
   }
 
+  get creationTime() {
+    return '';
+  }
+
   getChildren(): TektonNode[] {
     return this.children;
   }
+
+  getPipelineTasks(): string[] {
+    return this.pipelineTasks;
+  }
+
+  parseToolTip(): string {
+    return this.tooltip.substring(this.tooltip.indexOf("-") +1);
+  }
+
+  parseEventName(event: any): string {
+    return event.object.metadata.name.substring(0, event.object.metadata.name.lastIndexOf("-run"))
```

Comment:
I am not sure about this test, does the event name always have `-run` in its name ?

Reply:
This is a question that I have too. This way of associating the pipeline with the pipeline run makes me very uneasy but I wasn't sure what would be a better way. Any suggestions?

Reply:
@onyiny-ang there should be a label `tekton.dev/Pipeline` on the `PipelineRun` with the reference name of the Pipeline

Reply:
Ok thanks. I will check that out and see what i can figure out :+1: